### PR TITLE
[linstor] fixes in nodegroupconfigurations for DRBD install and linstor components update script

### PR DESCRIPTION
**Pull request by duckhawk**
## Description

We have some conditions in nodegroupconfigurations that not matches our code style. 
Potentially there can be problems with load dm-thin-pool module, that needed for lwm thin pool support.
Also, we need some additions in linstor components update script

## Why do we need it, and what problem does it solve?

This PR will fix conditions in nodegroupconfigurations that not matches our code style, force dm-thin-pool kernel module load and fix linstor components update script

## What is the expected result?

Some changes for unified code base, confidence about lvm-thin support and updated script for linstor components update.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: linstor
type: fix
summary: Fixes in nodegroupconfigurations for DRBD install and linstor components update script.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by duckhawk** [comment]
/e2e/abort static;Static;containerd;1.25 5807622623 5807622623-1-con-1-25 /sys/deckhouse-oss/install:pr5483 astra@192.168.199.168

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by duckhawk** [comment]
/e2e/abort static;Static;containerd;1.25 5808193176 5808193176-1-con-1-25 /sys/deckhouse-oss/install:pr5483 astra@192.168.199.138

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Review by AleksZimin — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by duckhawk** [comment]
/e2e/abort static;Static;containerd;1.25 5808501387 5808501387-1-con-1-25 /sys/deckhouse-oss/install:pr5483 astra@192.168.199.202

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've recognized your '/e2e/abort' command and started the workflow...

**Review by AleksZimin — APPROVED** [review]

File: modules/041-linstor/hack/update.sh [hunk 1]
```diff
@@ -39,6 +39,8 @@ while read name repo; do
       current_tag=$(curl -fLsS "https://api.github.com/repos/${shortrepo}/tags" | jq -r '.[] | .name' | sed -n 's|drbd-9|v9|p' | sort -V | tail -n1)
       # convert v9.X.X to 9.X.X
       drbd_version=${current_tag#*v}
+      # convert 9.X.X to 9XX
+      drbd_undotted_version=$(echo $drbd_version | sed 's/\.//g')
```
Comment (apolovov) [inline modules/041-linstor/hack/update.sh:43]:
```suggestion
      drbd_version_undotted=$(tr -d . <<< "$drbd_version")
```

File: modules/041-linstor/hack/update.sh [hunk 2]
```diff
@@ -59,4 +61,6 @@ echo "Applying changes:"
 (set -x; sed -e "$sed_regex" -i $targets)
 if [ -n "$drbd_version" ]; then
   (set -x; sed -e "/^      drbdVersion:/,/default:/{/^\([[:space:]]*default: \).*/s//\1\"${drbd_version}\"/}" -i openapi/values.yaml)
+  (set -x; sed 's/$version := "[0-9]\+\.[0-9]\+\.[0-9]"/$version := "'$drbd_version'"/' -i modules/007-registrypackages/images/drbd/werf.inc.yaml)
+  (set -x; sed 's/drbd[0-9]\+/drbd'$drbd_undotted_version'/' -i modules/041-linstor/templates/nodegroupconfiguration-drbd-install-*-like.yaml)
```
Comment (apolovov) [inline modules/041-linstor/hack/update.sh:65]:
```suggestion
  (set -x; sed 's/$version[[:space:]]+:=[[:space:]]+"[0-9]\+\.[0-9]\+\.[0-9]\+"/$version := "'$drbd_version'"/' -i modules/007-registrypackages/images/drbd/werf.inc.yaml)
  (set -x; sed 's/drbd[0-9]\+/drbd'$drbd_version_undotted'/' -i modules/041-linstor/templates/nodegroupconfiguration-drbd-install-*-like.yaml)
```

**Review by apolovov — APPROVED** [review]

**Review by AleksZimin — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [5501](https://github.com/deckhouse/deckhouse/pull/5501) to the branch [release-1.50](https://github.com/deckhouse/deckhouse/tree/release-1.50) successful!

**Pull request merged**